privval/signer_listener_endpoint.go

```diff
@@ -23,9 +23,9 @@ type SignerListenerEndpoint struct {
 	connectionAvailableCh chan net.Conn
 
 	timeoutAccept time.Duration
-	mtx           sync.Mutex
+	pingTimer     *time.Ticker
 
-	pingTimer *time.Ticker
+	instanceMtx sync.Mutex // Ensures instance public methods access, i.e. SendRequest
```

Comment:
I think we can just say

```
not concurrency safe
```

in the godoc comment and remove the instanceMtx

Reply:
This mutex is actually very important so external calls to SendRequest never mix with internal SendRequest(&PingRequest)

Reply:
Okay